### PR TITLE
regex added

### PR DESCRIPTION
**Pull request by Weedshaker**

### CODE REVIEW

File: src/es/components/controllers/fetchCss/FetchCss.js [hunk 1]
```diff
@@ -158,6 +158,7 @@ export default class FetchCss extends Shadow(WebWorker()) {
         style = await this.webWorker(FetchCss.cssNamespaceToVar, style, fetchCSSParam.namespace)
       }
     }
+    if (fetchCSSParam.replaces) style = await fetchCSSParam.replaces.reduce((style, replace) => this.webWorker(FetchCss.replace, style, replace.pattern, replace.flags, replace.replacement), style)
```
Comment (ASchmidt1024) [inline src/es/components/controllers/fetchCss/FetchCss.js:161]: This line is hard to understand but it does the work.

**Review by ASchmidt1024 — APPROVED** [review]
🎉 Nice work!

File: src/es/components/controllers/fetchCss/FetchCss.js (hunk 1 above)
Comment (Weedshaker) [reply src/es/components/controllers/fetchCss/FetchCss.js:161]: yes, reduce itself is kinda hard to read. any better proposals?

Comment (ASchmidt1024) [reply src/es/components/controllers/fetchCss/FetchCss.js:161]: I can't think of anything better atm 🤔 
```javascript
if (fetchCSSParam && fetchCSSParam.replaces) {
  style = await fetchCSSParam.replaces.reduce((style, replace) => this.webWorker(FetchCss.replace, style, replace.pattern, replace.flags, replace.replacement), style);
}
```

**Pull request merged**